README.md

```diff
@@ -1 +1,62 @@
-# stats-data
\ No newline at end of file
+# stats-data
+
+## Repo Overview
+
+This repo contains scripts to scrape various community data points.
+
+## Usage
+
+### Requirements
+
+| Script / Service | Requirements |
+| --- | --- |
+| discord | None |
+| github | None |
+| google | Custom Search Engine |
+| reddit | Registered reddit bot |
+| stackexchange | None |
+| twitter | Valid account |
+| youtube | Valid API Key |
+### Configuration
+
+Example `.env`
+
+```
+# Required
+CSE_API_KEY='m3DJU4hokX9haNZjvUWKkYnYUTrujAVD3FgWa9e'
+CSE_CX='dezZ22vMRorVt3eVa'
+TWITTER_AUTHORIZATION_HEADER='Bearer WXeXLCEECWy7hxVVXqA4gUybnzAWgj7gMfaFVnZhWvFDkWRU4pUNyb2Ph2s%3kRS7X4ae3gFKX2TfdJ597JxobCdxeQfA3aYURdASpUmw4fTDpf3'
+YOUTUBE_API_KEY='m3DJU4hokX9haNZjvUWKkYnYUTrujAVD3FgWa9e'
```

Comment:
Should these be public?

Reply:
these are real-looking examples, but I will update them to be less real-looking

Reply:
removed real-looking examples